newrelic/readme.md

```diff
@@ -8,36 +8,44 @@
 
-1. Click the **New Integration** button, and in the new field labelled **API key** enter the API key that you obtained in step 1. 
+1. Click the **New Integration** button. In the new field labelled **API key** enter the API key that you obtained in step 1, then click **Validate**.
 
-1. Optionally, select the individual New Relic modules for which you want metrics to appear in SignalFx. By default, SignalFx collects data from APM, Mobile, and Servers modules. 
+1. A message appears that says **Validated!**. If a different message appears, contact support@signalfx.com for help.
 
-1. Click **Save**. A message appears that says **Validated!**. If a different message appears, contact support@signalfx.com for help. 
+1. Next, create sync rules to control which metrics SignalFx will sync from New Relic. Select at least one New Relic module for which you want metrics to appear in SignalFx (APM, Mobile, or Servers). You can create sync rules for each module.
 
-Your New Relic account is now connected and ready to use. Data from New Relic will arrive at SignalFx within minutes. 
+1. Use typeahead to select at least one app to monitor; you can copy and paste results from the Sample Matches list. Click the **+** button to add each app to the list. When you have chosen all the apps you wish to monitor, click **Done**.
+
+1. (optional) Select metrics and stats to monitor. By default, all are monitored.
+
+1. (optional) Repeat steps 4-6 for each module.
+
+1. Click **Save**. The choices you made for each module are saved as individual sync rules. You can later reopen the New Relic integration to add, edit, or delete sync rules.
+
+Your New Relic account is now connected and ready to use. Data from New Relic will arrive at SignalFx within minutes.
 
 ### USAGE
 
 This section includes information that can help you build SignalFx charts from New Relic data.
 
 #### Example New Relic dashboards
 
-SignalFx provides example dashboards that you can use as a baseline for creating your own New Relic charts in SignalFx. You can import them to SignalFx by [clicking here to download the JSON file](https://github.com/signalfx/integrations/blob/master/newrelic/dashboards/Page_SignalFx%20collectd%20%2B%20New%20Relic.json). In SignalFx, open the menu in the top right-hand corner of the app next to your profile image, and selecting **Import dashboard group**. This will create a new dashboard group called "SignalFx + New Relic" that contains two example dashboards:
+This repository contains example dashboards that you can use as a baseline for creating your own New Relic charts in SignalFx. You can import them to SignalFx by [clicking here to download the JSON file](././dashboards/Page_SignalFx collectd + New Relic.json). In SignalFx, open the menu in the top right-hand corner of the app next to your profile image, and selecting **Import dashboard group**. This will create a new dashboard group called "SignalFx + New Relic" that contains two example dashboards:
```

Comment:
Not sure why this line changed - please revert to "SignalFx provides example dashboards" as we're most likely viewing this in-app, not in repo.